<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppPage</class>
 <widget class="QWidget" name="AppPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>App Page</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="installButton">
       <property name="text">
        <string>安装应用</string>
       </property>
       <property name="icon">
        <iconset theme="list-add"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="uninstallButton">
       <property name="text">
        <string>卸载应用</string>
       </property>
       <property name="icon">
        <iconset theme="list-remove"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="refreshButton">
       <property name="text">
        <string>刷新</string>
       </property>
       <property name="icon">
        <iconset theme="view-refresh"/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="appList">
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>应用名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>类型</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>版本</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>应用大小</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>文档大小</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Bundle ID</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>